_submissions/round-12/0/2015-02-06-about-filipe_fernandes.md

```diff
@@ -7,6 +7,6 @@ permalink: /2015/02/filipe_fernandes/
 tags:
   - Biography
 ---
-[Filipe Fernandes](http://ocefpaf.github.io/homepage/) is a physical oceanographers interested in education and computer literacy.  During the day Filipe works for the Southeast Coastal Ocean Observing Regional Association ([SECOORA](http://ocefpaf.github.io/secoora/) and at night writes a [blog](http://ocefpaf.github.io/python4oceanographers/) about the use of python in oceanography.
+[Filipe Fernandes](http://ocefpaf.github.io/homepage/) is a physical oceanographers interested in education and computer literacy.  During the day Filipe works for the Southeast Coastal Ocean Observing Regional Association ([SECOORA](http://ocefpaf.github.io/secoora/)) and at night writes a [blog](http://ocefpaf.github.io/python4oceanographers/) about the use of python in oceanography.
```

Comment:
1.  Best to line-wrap so that we can comment on specific lines more easily.
2.  Should be "oceanographer" (singular) rather than "oceanographers" (plural).
3.  We usually capitalize "Python".